synapse/handlers/account_data.py

```diff
@@ -171,7 +171,7 @@ def get_current_key(self, direction: str = "f") -> int:
         return self.store.get_max_account_data_stream_id()
 
     async def get_new_events(
-        self, user: UserID, from_key: int, **kwargs
+        self, user: UserID, from_key: int, **kwargs: Any
```

Comment:
what's this `kwargs` for?

Reply:
I think that `get_new_events` generally takes some random arguments which most things don't care about. I did verify that there *are* other arguments going into this call, I'll double check if we can tighten this up.

Reply:
I think this can be cleaned up, but I'd prefer to do it in a separate PR if that's OK!

Reply:
I resolved the other threads about this to keep the conversation in one spot.

Reply:
Sure, take this to another PR if you prefer.